tests/test_scrapers.py

```diff
@@ -44,7 +44,7 @@ def test_nationmedia_scraper(self):
         result_html = self.scraper.get_html_content(
             scrape_sites["nation"])
         items = result_html.find_all(
-            "div", class_="story-teaser medium-teaser")
+            "div", class_="story-teaser top-teaser")
```

Comment:
Hmm.. Why are we re-writing the scraping functions here instead of re-using what we have already?